### ISSUE TITLE
parseUserData is called before Login response

### ISSUE THREAD

**Issue by eybarta** [websanova/vue-auth#181]
Hi,

I'm successfuly using $auth.login({...})

The response from the webservice has all the user data I want o keep
inside $auth.user().. I disabled the fetchData() because I'm getting the
data from the initial login call and don't need to make another call for it..
I thought parseUserData is the method used to save the data to $auth.user,
but it seems to be called before the success() callback is called.

In short, my issue is, how do I populate $auth.user() with data I get in my
$auth.login() response?

thx

**Comment by websanova** [comment]
$auth.user() // get data
$auth.user(object) // set data.
